community/config/eggs/creating_a_custom_image.md

```diff
@@ -13,52 +13,62 @@ reading up if this all looks foreign to you.
 The most important part of this process is to create the [`Dockerfile`](https://docs.docker.com/engine/reference/builder/)
 that will be used by the Daemon. Due to heavy restrictions on server containers, you must setup this file in a specific manner.
 
-We try to make use of [Alpine Linux](https://alpinelinux.org) as much as possible for our images in order to keep their size down.
+In most images we try to use a [Debian based OS](https://www.debian.org) as much as possible for our images.
```

Comment:
I'd change this to "We try to use a [Debian based OS](https://www.debian.org) as much as possible for our images."